source/tasks/CreateOctopusRelease/CreateOctopusReleaseV3/task.json

```diff
@@ -59,6 +59,14 @@
             },
             "helpMarkDown": "Version 3 of this task has limited support for spaces. We recommend using version 4 of this task for a better experience."
         },
+        {
```

Comment:
This is our new project filter for tasks (these should be consistent across all tasks that show a Project dropdown)